board/flash.py

```diff
@@ -9,7 +9,7 @@
 if __name__ == "__main__":
   subprocess.check_call(f"scons -C {board_path}/.. -j$(nproc) {board_path}", shell=True)
 
-  serials = Panda.list()
+  serials = Panda.list_all()
```

Comment:
any way to avoid this? list is nice and short

Reply:
I don't know when this would be useful but it picks up the method over the list type when adding type hints
![image](https://github.com/commaai/panda/assets/4038174/fdb6ff09-84a1-4e3a-aa81-70ea3258a447)

Maybe there is a way to implement `list(Panda)` instead? that seems python-ish

Reply:
oh! I might have made a mistake. it might only affect the @staticmethod in PandaDFU, not the @classmethod in Panda

Reply:
moving the method to the end of the file is a workaround